src/components/stack-chart/index.js

```diff
@@ -99,21 +107,22 @@ class StackChartGraph extends React.PureComponent<Props> {
     );
   };
 
-  _onRightClickedCallNodeChange = (
-    callNodeIndex: IndexIntoCallNodeTable | null
-  ) => {
+  _onRightClickedCallNodeChange = (index: number | null) => {
```

Comment:
Please revert this name change, as it was clearer with `callNodeIndex`.

Reply:
Please add a comment explaining how this component only handles right clicked call nodes, but not right clicked markers, and add a reference to https://github.com/firefox-devtools/profiler/issues/2354.

This might be good to do in the chart props under `onRightClick: this._onRightClickedCallNodeChange`

Reply:
👍